### ISSUE TITLE
Add stress relief to more food an Drink items 

### ISSUE THREAD

**Issue by troopakoopa** [Reprieve-RP/feedback#66]
**What feature you want to improve**

Stress Relief

**How could this feature be improved? Please describe your idea as much as possible**

Add stress relief to the following

coffee 
joints
twerks 
snikkel 
food from burger shot

**Comment by JakeWalker1** [comment]
agreed, you gain so much stress from driving fast/doing stores and right now its only nuggets that reduce stress and i had to eat 40 to get rid of all my stress

**Comment by ixhal** [comment]
So we have a idea for this that will eventually be put in where medics can give you a special candy that will remove a lot of stress.
That being said its not in yet and I have just pushed a update to some food items to remove stress after eating/drinking them.
All the items you listed above now remove some amount of stress.
The highest stress remover currently in the city is the Bleeder burger which can remove anywhere from 10 - 16 stress.

The emotes will be worked on soon as well to remove stress slowly overtime.

**Comment by ixhal** [comment]
Certain emotes now relieve stress roughly every 30 seconds while you are playing the animation.
/e yoga is one that works, Will get a entire list at some point but most of the /sit /sitchair /lawnchair /chairf emotes all relieve stress, So does playing the guitar, /e sleep also.
Will leave this open until we add the sweets for medics.
